runtime/polkadot/src/lib.rs

```diff
@@ -1583,11 +1583,179 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(StakingBagsListMigrationV8, SessionHistoricalPalletPrefixMigration, SchedulerMigrationV3),
+	(
+		StakingBagsListMigrationV8,
+		SessionHistoricalPalletPrefixMigration,
+		SchedulerMigrationV3,
+		FixCouncilDepositMigration,
+	),
 >;
 /// The payload being signed in transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+/// A migration struct to fix some deposits in the council election pallet.
+///
+/// See more details here: https://github.com/paritytech/polkadot/issues/4160
+pub struct FixCouncilDepositMigration;
+
+impl FixCouncilDepositMigration {
+	fn execute(check: bool) -> frame_support::weights::Weight {
+		let accounts = vec![
+			[
+				172u8, 216, 115, 87, 170, 243, 188, 88, 181, 73, 157, 16, 214, 192, 48, 146, 239,
+				79, 168, 154, 196, 169, 194, 254, 122, 128, 246, 166, 205, 215, 176, 96,
+			],
+			[
+				164, 44, 232, 74, 83, 194, 159, 82, 232, 179, 36, 255, 38, 50, 209, 27, 131, 102,
+				14, 84, 172, 117, 220, 42, 68, 1, 156, 100, 180, 176, 241, 14,
+			],
+			[
+				144, 8, 9, 30, 45, 31, 178, 14, 109, 106, 70, 192, 214, 101, 145, 201, 176, 11,
```

Comment:
Sorry for being really annoying, but i would like that the plaintext addresses be commented above each hex string.

Reply:
good idea and already added, please check a handful manually.